semantic-core/src/Core/Core/Pretty.hs

```diff
@@ -99,21 +100,3 @@ prettyCore style = precBody . go . fmap name
         arrowL = case style of
           Unicode -> symbol "←"
           Ascii   -> symbol "<-"
-
-
-data Prec a = Prec
-  { precLevel :: Maybe Int
-  , precBody  :: a
-  }
-  deriving (Eq, Ord, Show)
-
-prec :: Int -> a -> Prec a
-prec = Prec . Just
-
-atom :: a -> Prec a
-atom = Prec Nothing
-
-withPrec :: Int -> Prec AnsiDoc -> AnsiDoc
-withPrec d (Prec d' a)
-  | maybe False (d >) d' = parens a
-  | otherwise            = a
```

Comment:
This is defined by `fused-syntax`, so we don’t need to define it ourselves any more.